src/components/learner-credit-management/tests/BudgetDetailPage.test.jsx

```diff
@@ -181,9 +181,10 @@ describe('<BudgetDetailPage />', () => {
   });
 
   it.each([
-    { displayName: null },
-    { displayName: 'Test Budget Display Name' },
-  ])('renders budget header data (%s)', ({ displayName }) => {
+    { displayName: null, assigned: false },
+    { displayName: 'Test Budget Display Name', assigned: false },
+    { displayName: 'Assigned Budget', assigned: true },
```

Comment:
nit: `assigned` -> `isAssigned` to make the name more explicit that we're working with a boolean.